xmipp

```diff
@@ -193,11 +193,14 @@ def modeConfig(overwrite: bool=False) -> Dict:
 	- (dict): Dictionary containig all config variables.
 	"""
 	configDict = {}
+	logger(getSectionMessage("Generating config file"), forceConsoleOutput=True)
```

Comment:
Do we want this logs always (example: when running mode all, as it calls mode config inside)?

Reply:
Also, generating config file is not always true, since it can be only read if overwrite is False and the file already exists. the function `writeConfig` is called regardless, but it might not change the file at all if the variables in it are all ok

Reply:
let me check all casuistics